container/libcontainer/helpers.go

```diff
@@ -92,29 +92,30 @@ func GetStats(cgroupManager cgroups.Manager, rootFs string, pid int, ignoreMetri
 	stats := toContainerStats(libcontainerStats)
 
 	// If we know the pid then get network stats from /proc/<pid>/net/dev
-	if pid > 0 {
-		if !ignoreMetrics.Has(container.NetworkUsageMetrics) {
-			netStats, err := networkStatsFromProc(rootFs, pid)
-			if err != nil {
-				glog.V(2).Infof("Unable to get network stats from pid %d: %v", pid, err)
-			} else {
-				stats.Network.Interfaces = append(stats.Network.Interfaces, netStats...)
-			}
+	if pid == 0 {
+		return stats, nil
+	}
+	if !ignoreMetrics.Has(container.NetworkUsageMetrics) {
+		netStats, err := networkStatsFromProc(rootFs, pid)
+		if err != nil {
+			glog.V(2).Infof("Unable to get network stats from pid %d: %v", pid, err)
```

Comment:
I realize this is following the pattern that's in the code already, but why log the error instead of returning it, here?